x/schema/keeper/msg_server_schema.go

```diff
@@ -63,6 +61,7 @@ func (k msgServer) CreateSchema(goCtx context.Context, msg *types.MsgCreateSchem
 		Schema:    &schema,
 		Timestamp: time.Now().Unix(),
 		IsActive:  true,
+		Metadata:  make(map[string]string),
```

Comment:
You should populate this with the metadata from the request.

Reply:
egg on my face, that would be nice lol